File: app/src/main/java/com/my/televip/virtuals/messenger/DispatchQueue.java
package com.my.televip.virtuals.messenger;

import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import de.robv.android.xposed.XposedHelpers;

public class DispatchQueue {

    Object dispatchQueue;

    public DispatchQueue(Object obj){
        dispatchQueue = obj;
    }

    public void postRunnable(Runnable runnable) {
        XposedHelpers.callMethod(dispatchQueue, Obfuscate.getMethodName("DispatchQueue", "postRunnable"), runnable);
    }

}

File: app/src/main/java/com/my/televip/virtuals/messenger/DispatchQueue.kt
package com.my.televip.virtuals.messenger

import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers

class DispatchQueue(@JvmField val dispatchQueue: Any?) {

    fun postRunnable(runnable: Runnable?) {
        val target = dispatchQueue ?: return
        XposedHelpers.callMethod(target, Obfuscate.getMethodName("DispatchQueue", "postRunnable"), runnable)
    }
}
